components/permalink_view/permalink_view.tsx

```diff
@@ -57,25 +56,26 @@ export default class PermalinkView extends React.PureComponent {
         this.mounted = false;
     }
 
-    doPermalinkEvent = async (props) => {
-        const postId = props.match.params.postid;
-        await this.props.actions.focusPost(postId, this.props.returnTo, this.props.currentUserId);
+    doPermalinkEvent = async (props: Props) => {
+        const postId = props?.match?.params.postid;
+        if (this.props.actions) {
```

Comment:
```suggestion
        if (this.props?.actions) {
```
It is because actions are optional in `Props`